fedimintd/src/bin/main.rs

```diff
@@ -18,23 +18,37 @@ use fedimint_wallet::Wallet;
 use fedimint_wallet::WalletConfigGenerator;
 use fedimintd::encrypt::*;
 use fedimintd::ui::run_ui;
+use fedimintd::ui::UiMessage;
 use fedimintd::*;
 use tracing_subscriber::prelude::*;
 use tracing_subscriber::EnvFilter;
 use tracing_subscriber::Layer;
 
 #[derive(Parser)]
 pub struct ServerOpts {
+    /// Path to folder containing federation config files
     pub cfg_path: PathBuf,
+    /// Password to encrypt sensitive config files
     #[arg(default_value = None)]
     pub password: Option<String>,
-    #[arg(default_value = None)]
-    pub ui_port: Option<u32>,
+    /// Port to run admin UI on
+    #[arg(default_value = "6000")]
+    pub ui_port: u32,
     #[cfg(feature = "telemetry")]
     #[clap(long)]
     pub with_telemetry: bool,
 }
 
+pub fn source_password(rpcpassword: Option<String>) -> String {
+    match rpcpassword {
+        None => rpassword::prompt_password("Enter gateway password:").unwrap(),
```

Comment:
No point for interactive password prompt. We should only support `FEDIMINT_PASSWORD` env variable.